cmd/juju/space/space.go

```diff
@@ -40,7 +40,7 @@ type SpaceAPI interface {
 
 	// RemoveSpace removes an existing Juju network space, transferring
 	// any associated subnets to the default space.
-	RemoveSpace(name string) error
+	RemoveSpace(name string, currentModel string, force bool, dryRun bool) (error, bool)
```

Comment:
Alternative implementation:
return the RemoveSpaceResult or a similar struct and build the error message on the CMD level.